junqo_front/config/.env.ci

```
@@ -0,0 +1,7 @@
+# Description: Environment file for the front-end in CI/CD pipelines
+# For more information, see the documentation:
+# - [backend configuration](http://doc.junqo.fr/developer/frontend/#configuration)
+
+# The url to the backend api where the front-end will send requests
+# Default: "http://localhost:4200/api/v1"
+API_URL='http://localhost:4200/api/v1'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)